src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -1800,6 +1800,108 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_model_shape) {
     test_case.run();
 }
 
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_shape15_end_1) {
+    // shape 15 tests
+    auto model = convert_model("shape15.onnx");
+
+    Inputs inputs;
+    inputs.emplace_back(
+        ov::test::NDArray<float, 3>({{{1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}},
+                                     {{1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}},
+                                     {{1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}, {1, 1, 1, 1, 1}}})
+            .get_vector());
+
+    auto test_case = ov::test::TestCase(model, s_device);
+    test_case.add_multiple_inputs(inputs);
+    test_case.add_expected_output<int64_t>({3});
+    test_case.run();
+}
+
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_shape15_end_negative_1) {
+    // shape 15 tests
+    auto model = convert_model("shape15.onnx");
```

Comment:
I have a strong feeling it is copy-paste issue :) Could you check filenames? I see you introduced several test models, but don't use them.

Reply:
sorry, I have corrected it